run.go

```diff
@@ -326,38 +329,96 @@ func Log(c chan *SimulationStatus) DomainManipulator {
 	}
 }
 
-func (d *InMAP) checkOutputNames(n ...string) error {
+func removeDuplicates(s []string) []string {
+	result := []string{}
+	seen := map[string]string{}
```

Comment:
Maps always need to be initialized using the make comand (e.g., make(map[string]string) ) before using them. Changing this line to seen := make(map[string]string) may fix your problem